src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_params.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
```

Comment:
Please do not disable pylint rules in a file scope. Make the follow the style instead of bending the rules. There are existing pylint disable statement in the legacy code base, we're in the process of retiring them.